[16.0][IMP] queue_job: add error handler (#2)
trisdoan/queue: queue_job/readme/USAGE.rst

* retries 10 to 15 postponed 30 seconds later
* all subsequent retries postponed 5 minutes later

**Job function: Error Handler**

> The *Error Handler* is actions executed when job fails

Reviewer: ```suggestion
The *Error Handler* is a method executed whenever the job fails
```

It's configured similarly to Related Action

There is an OOTB handler: _call_webhook, which calls a webhook with configurable information.

> The webhook URL is configured in System parameters via key **queue_job.webhook_url**

Reviewer: ```suggestion
```

Reviewer: not anymore

> Example of using _call_api:

Reviewer: I think you can give the slack example to make it even more appealing/useful out of the box (with a webhook url that looks real but with tokens that you manually changed to XXX...)
